ui/components/data-sources/AddDataSourceDialog.js

```diff
@@ -8,9 +8,16 @@ import GetDataSources from "../../graphql/GetDataSources.graphql";
 const INITIAL_STATE = {
     name: "",
     type: DataSourceType.InMemory,
-    options: {
+    inMemoryOptions: {
         timestampData: true
     },
+    postgresOptions: {
+        url: "",
+        port: 5432,
```

Comment:
Should it not be a parameter?

Reply:
It is, thats just the default value (Postgres default port)

Reply:
Yep, but it could be changed. Also, it should be better if was defined by a const do you don't think? Something as follows. 

```javascript
const POSTGRES_PORT = process.env.POSTGRES_PORT ; // it should came from a property setup
{....}
port: POSTGRES_PORT,
```